src/patternfly/components/FormControl/examples/FormControl.md

```diff
@@ -119,43 +119,43 @@ cssPrefix: pf-v6-c-form-control
 
 ### Textarea
 ```hbs
-{{#> form-control controlType="textarea" form-control--attribute='name="textarea-standard" id="textarea-standard" aria-label="Standard textarea example"'}}
+{{#> form-control controlType="textarea" form-control--IsTextarea="true" form-control--attribute='name="textarea-standard" id="textarea-standard" aria-label="Standard textarea example"'}}
```

Comment:
This could always be a followup, but would be good to just leverage the `controlType` to determine text-area things rather than also needing to pass IsTextarea